templates/_asset-card-image.html

```diff
@@ -76,14 +77,16 @@
         <p>
           <strong>Asset details</strong>
         </p>
+         <hr class="p-rule" />
+        <p>
+          Salesforce Campaigns: <ul class="p-list--divided">{% for campaign in asset.salesforce_campaigns %}<li  class="p-list__item">{{ campaign.name }}</li>{% if not loop.last %} {% endif %}{% endfor %}</ul>
```

Comment:
It isn't clear to me what the purpose of this is: `{% if not loop.last %} {% endif %}`